static/js/protein/chart.ts

```diff
@@ -31,6 +38,27 @@ export interface ProteinStrData {
   value: string;
 }
 
+// types and interfaces for protein-protein interaction chart
+type NodeID = string | number;
```

Comment:
Sorry! I had copied the line 508 example data from an old implementation I inherited from Sam/Suhana where they ID'd with numbers.  I now id with {protein}_{species} (e.g. P53_HUMAN).  I'll update 508 with the new IDs.  And I think just in case this interface gets reused for some other graph viz in the future perhaps we can leave the numerical ID support in there?

Reply:
currently the Node interface on line 44 is just used here right? you can just make id type to be string .. I don't think it's necessary to create a separate interface NodeID.

Reply:
sounds good!